### PR TITLE
Add Sandbox troubleshooting notes to the README

### PR DESCRIPTION
**Pull request by JimRottinger**
Title. Just added some sandbox troubleshooting notes to the README since a lot of people still seem to want to use these.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,51 @@
 
 ----
 
-**A Note on Support**
+## Sandboxed Custom Visualizations Troubleshooting
+
+While the visualizations in this repository are deprecated and therefore not officially supported by Sandboxed Custom Visualizations (aka Custom Viz V2), it is possible to make them work. All it takes is understanding how to source the file and include the correct dependencies.
```
Comment (RichardCzechowski) [inline README.md]: Link to main docs from "Sandboxed Custom Visualizations"?

File: README.md [hunk 2]
```diff
@@ -10,7 +10,51 @@
...
+
+#### Sourcing the File
+
+One of the main features of Custom Viz V2 is being able to include the URL of a visualization. This is diserable because both Looker and its customers can self-host thier visualizations and pushing an update to that file will automatically update all visualizations in the Looker instance. To make the visualizations in this repository work with that format, you have 2 options:
```
Comment (RichardCzechowski) [inline README.md]: `s/thier/their`

File: README.md [hunk 3]
```diff
@@ -10,7 +10,51 @@
...
+
+###### Option 1 (**Unsupported** but easiest): Use RawGit to create a production URL
+
+Github stores files as text, so just copy-and-pasting the github URL into your visualization definition will not be enough. It needs the correct content headers to be able to execute as JavaScript. There is an third-party app called [RawGit](rawgit.com) that will take GitHub files and host them with the correct headers.
+
+To use it, grab the Github URL of the JS file you want to use, such as [the spiderweb vis](https://github.com/looker/visualization-api-examples/blob/master/examples/highcharts_example/highcharts_spiderweb.js). Copy and paste the URL into RawGit and then grab the Production URL that it gives you. Use this link in the `main` field of your visualization definition.
+
+**Please Note** that we do not officially support this method as we do not own or partner with RawGit, so there is no guarantee of the uptime and security of the visualization files.
```
Comment (RichardCzechowski) [inline README.md]: Another caveat is that we reserve the right to close down this repo in the future if it continues to be troublesome.

File: README.md [hunk 4]
```diff
@@ -10,7 +10,51 @@
...
+
+###### Option 2 (Safest): Copy the contents of the file into the instance's `looker/plugins/visualizations` folder
```
Comment (RichardCzechowski) [inline README.md]: Hmm. Only works for On-Prem. Hosted customers can't and shouldn't be accessing their instance. Not to mention that this requires the old custom vis license in addition to the new one. This probably should not be documented

**Comment by JimRottinger** [comment]
@RichardCzechowski re-review please!

File: README.md [hunk 5]
```diff
@@ -10,7 +10,45 @@
...
+
+###### Hosting the JS file: Use RawGit to create a production URL
+
+Github stores files as text, so just copy-and-pasting the github URL into your visualization definition will not be enough. It needs the correct content headers to be able to execute as JavaScript. There is an third-party app called [RawGit](https://rawgit.com/) that will take GitHub files and host them with the correct headers.
+
+To use it, grab the Github URL of the JS file you want to use, such as [the spiderweb vis](https://github.com/looker/visualization-api-examples/blob/master/examples/highcharts_example/highcharts_spiderweb.js). Copy and paste the URL into RawGit and then grab the Production URL that it gives you. Use this link in the `main` field of your visualization definition.
+
+**Please Note** that we do not officially support this method as we do not own or partner with RawGit, so there is no guarantee of the uptime and security of the visualization files.
```
Comment (RichardCzechowski) [inline README.md]: Wouldn't be a bad idea to recommend a second method such as uploading to AWS that's a method we would actively condone.

File: README.md [hunk 6]
```diff
@@ -10,7 +10,45 @@
...
+
+#### Understanding Dependencies
+
+The biggest difference between custom viz V1 and V2 is that the visualizations in V2 execute inside of a iframe. This means the visualizations do not have access to Looker global variables such as jQuery ($) and D3. A lot of the visualizations in this repository assume those globals will be there, so we have to include them as dependencies. You can add dependencies by opening up the advanced options when defining your visualization.
```
Comment (RichardCzechowski) [inline README.md]: "an iframe"

File: README.md [hunk 7]
```diff
@@ -10,7 +10,45 @@
...
+
+**Dependencies to Include in ALL Visualizations in this Repository**
+
+1. jQuery - `https://code.jquery.com/jquery-3.3.1.min.js`
+2. Underscore - `https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js`
+3. Looker Vis Utils -`'https://looker-custom-viz-a.lookercdn.com/master/v1_common.js`
+
+**Dependencies you *might* neeed**
+
+1. D3 - `https://cdnjs.cloudflare.com/ajax/libs/d3/4.13.0/d3.js`
```
Comment (RichardCzechowski) [inline README.md]: I believe the old version of d3 that we used to provide was 3.5.17

File: README.md [hunk 8]
```diff
@@ -10,7 +10,45 @@
...
+2. D3v4 - `https://cdn.rawgit.com/looker/visualization-api-examples/master/examples/common/d3.v4.js`
+3. Highcharts - `https://looker-custom-viz-a.lookercdn.com/master/highcharts.js`
```
Comment (RichardCzechowski) [inline README.md]: Maybe this could be a specific version of highcharts served from their own CDN?

**Pull request closed without merging**